solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-10.adoc

```diff
@@ -26,6 +26,11 @@ Before starting an upgrade to this version of Solr, please take the time to revi
 
 // TODO add similar text that previous releases have at this spot.
 
+=== Solr CLI and Scripts
+
+The Solr CLI has gone through some signficant rennovations to reduce technical debt, and function more consistently and predictably.
```

Comment:
```suggestion
The Solr CLI has gone through some significant renovations to reduce technical debt, and now functions more consistently and predictably. Most notably, CLI commands now use unix-style options, e.g. `--help` instead of `-help`.
```